### PR TITLE
binance: don't enable BTC/USDT instrument without user's knowledge

### PR DESCRIPTION
**Pull request by lrascao**
If not instruments were enabled refrain from trying to be clever and
auto-enabling BTC/USDT.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

- [X] go test ./... -race
- [X] golangci-lint run
- [ ] Test X

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/binance/binance_wrapper.go [hunk 1]
```diff
@@ -264,59 +264,10 @@ func (b *Binance) Run() {
 	}
 
 	forceUpdate := false
```
Comment (shazbert) [inline exchanges/binance/binance_wrapper.go:266]: you can remove this as well.

**Review by shazbert — CHANGES_REQUESTED** [review]
Just one thing. I think time has sufficiently passed to warrant this removal.

**Comment by thrasher-** [comment]
NOTE: This is currently the way we perform config pair format upgrades whenever a delimiter is required to parse pairs properly. See https://github.com/thrasher-corp/gocryptotrader/pull/762 as an example. By default, this behaviour is enabled and provides warnings to the user whenever such a change to the config takes place. Although this one was a fix introduced years ago, there will be future ones made. A config setting moving forward would be better for users who don't require this functionality, but for an out of box experience we want it to be as pain free for users as possible when it comes to things like this enabled by default. Another thing to consider is the config loader will assign a random pair if a user has enabled an asset, but hasn't filled in any enabled pairs on GCT start-up.

**Review by shazbert — APPROVED** [review]
Changes ACK.

**Review by gloriousCode — APPROVED** [review]
This scenario seemingly can only come up if you're purposely storing pairs in a different format. Using `config_example.json`, (our basic backwards-compatible config) doesn't trigger this piece of code. So in that sense its fine to remove this specific instance of config pair storage fixing as it doesn't impact config backwards compatibility. tACK.

**Review by thrasher- — APPROVED** [review]

**Pull request merged**